### ISSUE TITLE
Warn: Failed to get best common ancestor commit despite not configuring "UseMostCommonAncestor"

### ISSUE THREAD

**Issue by thomasheslinbmll** [jfrog/frogbot#782]
### Describe the bug

I am seeing the following warning: ```Failed to get best common ancestor commit between source branch: <FEATURE-BRANCH> and target branch: master, defaulting to target branch commit. Error: reference not found```

I can see this was introduced last week in #763, which states:

> To use this option you can:
> * Specify it as an env var: JF_USE_MOST_COMMON_ANCESTOR_AS_TARGET
> * Specify it in Frogbot config: useMostCommonAncestorAsTarget (at Git section)

I am not seeing `JF_USE_MOST_COMMON_ANCESTOR_AS_TARGET` nor using `useMostCommonAncestorAsTarget`.

### Current behavior

In my Frogbot workflow I am seeing:

```
2024-11-06T23:25:54.9660118Z 23:25:54 [Info] <REPO> repository downloaded successfully. Starting with repository extraction...
2024-11-06T23:25:54.9664650Z 23:25:54 [Info] Extracted repository successfully
2024-11-06T23:25:55.0776344Z 23:25:55 [Debug] Repository download completed
2024-11-06T23:25:55.0777605Z 23:25:55 [Debug] Using most common ancestor commit as target branch commit
2024-11-06T23:25:55.2589102Z 23:25:55 [Debug] Setting timeout for go-git to 120 seconds ...
2024-11-06T23:25:55.2593431Z 23:25:55 [Warn] Failed to get best common ancestor commit between source branch: another_update and target branch: master, defaulting to target branch commit. Error: reference not found
2024-11-06T23:25:55.2595479Z 23:25:55 [Info] Scanning target branch...
2024-11-06T23:25:55.2596430Z 23:25:55 [Debug] Sending HTTP GET request to: https://<COMPANY>.jfrog.io/xray/api/v1/system/version
```

I do not have a frogbot-config.yml for this repo and the Frogbot step looks like:

```yml
- uses: jfrog/frogbot@v2
  with:
    oidc-provider-name: github-dev
    oidc-audience: jfrog-github-dev
  env:
    JF_DEPS_REPO: pypi-virtual
    JF_GIT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    JF_MIN_SEVERITY: Low
    JF_URL: "https://<COMPANY>.jfrog.io"
    JFROG_CLI_LOG_LEVEL: DEBUG
```

### Expected behavior

The original checkout logic to behave.

### JFrog Frogbot version

Frogbot version: 2.23.2

### Package manager info

Python 3.11 (requirements.txt)

### Git provider

GitHub

### JFrog Frogbot configuration yaml file

N/A

### Operating system type and version

Ubuntu 22

**Comment by thomasheslinbmll** [comment]
@ismaelhamed one option is to use [v2.23.1](https://github.com/jfrog/frogbot/releases/tag/v2.23.1) in the meantime.

Has anyone at JFrog taken a look at this?

**Comment by eyalk007** [comment]
Hello @thomasheslinbmll and @ismaelhamed,

Thank you for using Frogbot and for bringing this issue to our attention. We truly appreciate your feedback.

Our team is actively working on a solution, and we will keep you updated as soon as a fix is available.

Thank you for your patience!

**Comment by eyalk007** [comment]
Hi @thomasheslinbmll and @ismaelhamed,

I wanted to let you know that the bug you reported has been resolved in the latest version (`v2.23.8`) of Frogbot. Please let us know if you encounter any further issues😊

Thank you!

**Comment by thomasheslinbmll** [comment]
Great, thank you!
